### DISCUSSION TITLE
Tiling for 360vr3D

### DISCUSSION THREAD

**Discussion by proofoftrust21** [thygate/stable-diffusion-webui-depthmap-script#203] [General]
Hello,

Do you know a way to join the ends of the images in VR360, like the tiling option in SD?
When generating a top/bottom stereoscopic image for example.
Thanks in advance.

**Comment by thygate** [comment]
Check this plugin : https://github.com/tjm35/asymmetric-tiling-sd-webui

**Reply by proofoftrust21** [reply]
Very cool, thank you!
